### PR TITLE
mood.coffee

### PR DESCRIPTION
**Pull request by max-sec**
dumb script for dumb malk

### CODE REVIEW

File: scripts/mood.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Description:
+#   Hubot will flip a fucking coin.
```
Comment (skalnik) [inline scripts/mood.coffee:2]: This isn't true anymore :0

File: scripts/mood.coffee [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Commands:
+#   mood - declare your daily mood
+#
+# Author:
+#   max-sec
+
+module.exports = (robot) ->
+  robot.respond /(coinflip)/i, (msg) ->
+    result = msg.random ["https://usercontent.irccloud-cdn.com/file/DbMM89Wi/mood", "https://usercontent.irccloud-cdn.com/file/3g41hAJc/signal-2022-09-22-08-37-39-817.jpg"]
```
Comment (sshirokov) [inline scripts/mood.coffee:18]: Plz newlines

File: scripts/mood.coffee [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+#   Hubot will flip a fucking coin.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   mood - declare your daily mood
```
Comment (sshirokov) [inline scripts/mood.coffee:11]: This seems like it tells you what ur mood is, not u declaring what ur mood is

File: scripts/mood.coffee [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Author:
+#   max-sec
+
+module.exports = (robot) ->
+  robot.respond /(coinflip)/i, (msg) ->
+    result = msg.random ["https://usercontent.irccloud-cdn.com/file/DbMM89Wi/mood", "https://usercontent.irccloud-cdn.com/file/3g41hAJc/signal-2022-09-22-08-37-39-817.jpg"]
+    msg.send "And the result is: #{result}
```
Comment (sshirokov) [inline scripts/mood.coffee:19]: Missing newline at EOF

**Review by sshirokov — CHANGES_REQUESTED** [review]
Almost

File: scripts/mood.coffee (hunk 3 above)
Comment (sshirokov) [reply scripts/mood.coffee:11]: !?!??!!

File: scripts/mood.coffee [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Commands:
+#   mood - declare your daily mood
+#
+# Author:
+#   max-sec
+
+module.exports = (robot) ->
+  robot.respond /(mood)/i, (msg) ->
+    result = msg.random ["shorturl.at/ceiVZ", "shorturl.at/hiQRT"]
```
Comment (sshirokov) [inline scripts/mood.coffee:18]: if they're urls, they should start with the protocol - and also why a shortner, wouldn't that just make it even more likely to eventually fail?

Comment (max-sec) [reply scripts/mood.coffee:18]: Temp use for the lols. I'll host them myself and update later

File: scripts/mood.coffee (hunk 4 above)
Comment (max-sec) [reply scripts/mood.coffee]: updated in merge after this

File: scripts/mood.coffee (hunk 2 above)
Comment (max-sec) [reply scripts/mood.coffee]: updated in merge after this

File: scripts/mood.coffee (hunk 3 above)
Comment (max-sec) [reply scripts/mood.coffee:11]: It is declaring your mood, but I can adjust language if wanted

**Pull request merged**